src/radix_connect/mobile_connect_request.rs

```diff
@@ -0,0 +1,19 @@
+use crate::prelude::*;
+
+#[derive(Debug, uniffi::Enum)]
+pub enum MobileConnectRequest {
```

Comment:
Sample values for these models would be nice to have

Reply:
Yes! ALWAYS SampleValues for ALL types :) and then test:

(here, example: `SargonBuildInformation`)
```rust

#[cfg(test)]
mod tests {
    use super::*;

    #[allow(clippy::upper_case_acronyms)]
    type SUT = SargonBuildInformation;

    #[test]
    fn equality() {
        assert_eq!(SUT::sample(), SUT::sample());
        assert_eq!(SUT::sample_other(), SUT::sample_other());
    }

    #[test]
    fn inequality() {
        assert_ne!(SUT::sample(), SUT::sample_other());
    }
}

```